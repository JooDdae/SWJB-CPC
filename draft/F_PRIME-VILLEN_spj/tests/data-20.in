593332958813659760312013037598112311171790785219619893349691613723742894951211565795213067592342434789274188155638537186128119871931318758619127724369714178734993745722876373972179602969918311338937679587919178315239538447661174138771123316716216967781766072803966169614447226726933187837793615131871097533342119413733178720691999719853749093853228767935113725363534051187726711491103
