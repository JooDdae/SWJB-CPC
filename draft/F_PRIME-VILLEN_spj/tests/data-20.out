187
17 2 2 2 10 3 152 2 120 966 197 435 782 0 1 2 0 0 0 4 0 4 155 1062 1187 24 2 2 665 0 18 2 4 1 2 4 589 662 1 0 0 745 1179 439 17 1 2 582 15 24 1 4 13 152 3 103 147 4 284 60 300 8 2 0 451 771 1131 4 1 14 896 13 138 2 95 2 4 88 340 831 25 1 0 22 786 899 23 0 0 2 2 24 2 4 19 1184 157 0 4 23 0 3 52 3 2 1057 121 0 4 13 2 151 0 0 1 2 2 0 19 257 895 988 854 1 1011 121 894 604 1 1 19 57 2 2 0 1095 4 22 2 18 3 0 451 29 4 3 2 2 2 82 8 13 2 4 2 2 0 138 312 303 4 8 147 4 656 2 147 340 875 3 0 0 2 4 1 3 2 827 559 0 151 1 19 0 35 0 27 